python/mxnet/gluon/rnn/rnn_layer.py

```diff
@@ -286,6 +286,9 @@ class RNN(_RNNLayer):
     Inputs:
         - **data**: input tensor with shape `(sequence_length, batch_size, input_size)`
           when `layout` is "TNC". For other layouts dimensions are permuted accordingly.
+          Be aware that a `transpose` operation with a ndarray results in a new allocation of
```

Comment:
Looks too long. Just adding a short sentence a long the line of "TNC is faster" should be enough.

Reply:
That was my original formulation but then I realized that if users did a ndarray.transpose to TNC before loading the data then it would be pointless advice 😄 And I didn't want to either explicitly say that numpy is faster at transposing since it is copy-free.

Reply:
it's still a copy when you load a view of numpy array into ndarray. User need to read data so that it comes in as TNC.

Reply:
that's true, What I mean is:
read data into numpy NTC -> nd array NTC -> nd array TNC  = 2 copies
read data into numpy NTC -> numpy TNC -> nd array TNC = 1 copy